lib/wickedbase.pm

```diff
@@ -43,6 +43,12 @@ sub get_ip {
     elsif ($args{type} eq 'sit1') {
         return $args{is_wicked_ref} ? '2001:0db8:1234::000e' : '2001:0db8:1234::000f';
     }
+    elsif ($args{type} eq 'tunl1') {
+        return $args{is_wicked_ref} ? '3.3.3.10' : '3.3.3.11';
+    }
+    elsif ($args{type} eq 'br0') {
+        return $args{is_wicked_ref} ? '10.0.2.10' : '10.0.2.11';
```

Comment:
isn't this suppose to be part of another commit?

Reply:
yes, sorry, let's merge https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/5396  first and then I'll rebase this on master.

Reply:
both PRs were not merged yet, better delete code from here and add it to #5396. so ALL code related to bridge will be in one commit

Reply:
done. Anyway, I might have to solve a git conflict